InterfaceParser/parsers/parse_error.py

```diff
@@ -0,0 +1,11 @@
+"""Parse error.
+
+"""
+
+
+class ParseError(Exception):
+    """Parse error.
+
+    This exception is raised when XML contains errors and can't be parsed.
+
+    """
```

Comment:
Shouldn't there be some error handling in this class?

Reply:
this error appear only in case when when the incorrect/corrupted MOBILE_API.xml provided to parser. In such case parsed won't be able to extract correct model, so it operation will be interrupted with exception. In all other cases this exception will not be raised. Or do you have any idea how should we handling such exception?